packages/templates/cra-template-desktop-viewer/template.json

```diff
@@ -68,6 +68,11 @@
       "start": "npm run build:backend && run-p \"start:frontend\" \"electron:debug\"",
       "start:frontend": "react-scripts start"
     },
+    "pnpm": {
```

Comment:
if youre going to add the override, then i would recommend changing the recommended approach in the readme from using npx to pnpm dlx, take a look at docs for that

Reply:
changed readmes